news/get-authors.py

```diff
@@ -2,8 +2,10 @@
 
 from subprocess import check_output
 
+last_tag = check_output(["git", "describe", "--abbrev=0"])
 authors = {}
-output = check_output(["git", "log", "--format=%ae\t%an"], universal_newlines=True)
+log_range = last_tag.decode("utf-8").strip() + "..HEAD"
```

Comment:
You could also use `universal_newlines=True` to convert the bytes to a string. This is what is used by the other `check_output` call.

Reply:
Thanks for hint. I will do this.

Reply:
Still need to strip for '\n'.